### PR TITLE
[HOTFIX - Merge with gitflow] Fix state filter and update presidential downloads list (Deploy 2/1/2020)

### LINKED ISSUE #3511: Update presidential candidate downloads [Due 2/1/2020]
### Summary

**What we're after:**
We need to update the candidate list to include those candidates that meet our $100,000 threshold. We need to update `CONTRIBUTION TOTALS BY STATE` to `INDIVIDUAL CONTRIBUTIONS BY STATE` in each accordion. 

image

image

### Completion criteria
- [ ] Update candidate list
- [x] Update section heading in each accordion: `CONTRIBUTION TOTALS BY STATE` to `INDIVIDUAL CONTRIBUTIONS BY STATE`
- [x] Update OVERALL CANDIDATE SUMMARY links from `2020_overall_summary_now.csv` to `2020_overall_summary.csv` for each election year
- [x] bold `other than the candidate` in each accordion
-  [ ] Deploy HOTFIX

### LINKED ISSUE #3512: Filters not working.
@jbucelato commented on [Fri Jan 31 2020](https://github.com/fecgov/FEC/issues/9498)

There is a candidate missing from the list, Tonya Millis, https://www.fec.gov/data/candidates/?party=LIB&name=IN&is_active_candidate=true. Also, I selected a specific state and I getting candidates from other states,
The candidate does appear correctly here, https://www.fec.gov/data/candidates/house/?election_year=2020&election_full=True&party=LIB&name=IN&is_active_candidate=true, however, candidates from other states appear as well.

Thanks for looking into this.

Jason

---

@PaulClark2 commented on [Fri Jan 31 2020](https://github.com/fecgov/FEC/issues/9498#issuecomment-580870110)

The state filter seems to be applying a name filter instead of a state filter: https://www.fec.gov/data/candidates/?name=IN 

image

The state filter seems to be applying a name filter instead of a state filter:
https://www.fec.gov/data/candidates/house/?election_year=2020&election_full=True&party=LIB&name=IN&is_active_candidate=true

image

### PR DESCRIPTION
**Pull request by patphongs**
## Summary

- Resolves #3511 
- Resolves #3512 
_This PR fixes the datatable state filters. Also adds updates for presidential downloads after 1/31 filings._

## Impacted areas of the application
List general components of the application that this PR will affect:

- All datatables that use the state filter
- Presidential downloads in the candidates tab: http://localhost:8000/data/browse-data/?tab=candidates

## How to test
- [ ] Checkout branch
- [ ] Go to Candidates tab: http://localhost:8000/data/browse-data/?tab=candidates. Check accordions under `Presidential candidates`
- [ ] Ensure the candidate list has been updated according to this doc: https://docs.google.com/spreadsheets/d/1_oOGQL7jF5bP1VYDuBh3KFRKXRCWPAvwkvb24kJ-CcM/edit#gid=894303291 🔒 
- [ ] Ensure section heading in each accordion: CONTRIBUTION TOTALS BY STATE is updated to INDIVIDUAL CONTRIBUTIONS BY STATE
 - [ ] Ensure OVERALL CANDIDATE SUMMARY links from [YEAR]_overall_summary_now.csv to [YEAR]_overall_summary.csv for each election year
- [ ] Ensure "other than the candidate" text is bolded.
- [ ] Check a datatable page and make sure that the state filter is working. Ex: http://localhost:8000/data/candidates/?party=LIB&state=IN&is_active_candidate=true
____

### CODE REVIEW

File: fec/data/templates/partials/browse-data/candidates.jinja [hunk 1]
```diff
@@ -81,7 +81,7 @@
                   <li><a href="/files/bulk-downloads/Presidential_Map/2008/Contributions_by_3Zip.csv">2008</a></li>
                 </ul>
               </div>
-              <h6 class="t-no-rules">Contribution totals by state</h6>
+              <h6 class="t-no-rules">Individual contribution totals by state</h6>
```
Comment (PaulClark2) [inline fec/data/templates/partials/browse-data/candidates.jinja]: @patphongs "Individual contribution totals by state" should be "Individual contributions by state"  We don't need "totals."

File: fec/data/templates/partials/browse-data/candidates.jinja [hunk 2]
```diff
@@ -160,8 +160,8 @@
           <div class="js-accordion accordion--neutral" data-content-prefix="all-presidential-democratic-candidates">
             <button type="button" class="js-accordion-trigger accordion__button">All Democratic presidential map candidates</button>
             <div class="accordion__content">
-              <p>This data summarizes financial information disclosed by Democratic presidential candidates who have reported at least $100,000 in contributions from individuals other than the candidate.</p>
-              <h6 class="t-no-rules">Contribution totals by state</h6>
+              <p>This data summarizes financial information disclosed by Democratic presidential candidates who have reported at least $100,000 in contributions from individuals <strong>other than the candidate</strong>.</p>
+              <h6 class="t-no-rules">Individual contribution totals by state</h6>
```
Comment (PaulClark2) [inline fec/data/templates/partials/browse-data/candidates.jinja]: Same as above "Individual contribution totals by state" should be "Individual contributions by state"  We don't need "totals."

File: fec/data/templates/partials/browse-data/candidates.jinja [hunk 3]
```diff
@@ -241,8 +241,8 @@
           <div class="js-accordion accordion--neutral" data-content-prefix="all-presidential-republican-candidates">
             <button type="button" class="js-accordion-trigger accordion__button">All Republican presidential map candidates</button>
             <div class="accordion__content">
-              <p>This data summarizes financial information disclosed by Republican presidential candidates who have reported at least $100,000 in contributions from individuals other than the candidate.</p>
-              <h6 class="t-no-rules">Contribution totals by state</h6>
+              <p>This data summarizes financial information disclosed by Republican presidential candidates who have reported at least $100,000 in contributions from individuals <strong>other than the candidate</strong>.</p>
+              <h6 class="t-no-rules">Individual contribution totals by state</h6>
```
Comment (PaulClark2) [inline fec/data/templates/partials/browse-data/candidates.jinja]: Same as above "Individual contribution totals by state" should be "Individual contributions by state"  We don't need "totals."

File: fec/data/templates/partials/browse-data/candidates.jinja [hunk 4]
```diff
@@ -322,10 +322,10 @@
           <div class="js-accordion accordion--neutral" data-content-prefix="individual-presidential-candidates">
             <button type="button" class="js-accordion-trigger accordion__button">Individual presidential map candidates</button>
             <div class="accordion__content">
-              <p>This data summarizes financial information disclosed by 2020 presidential candidates who have reported at least $100,000 in contributions from individuals other than the candidate.</p>
+              <p>This data summarizes financial information disclosed by 2020 presidential candidates who have reported at least $100,000 in contributions from individuals <strong>other than the candidate</strong>.</p>
               {% for candidate_id, candidate in load_presidential_map_candidate_data.items() %}
               <h4><a href="/data/candidate/{{ candidate_id }}/">{{ candidate.name }}</a></h4>
-              <h6 class="t-no-rules">Contribution totals by state</h6>
+              <h6 class="t-no-rules">Individual contribution totals by state</h6>
```
Comment (PaulClark2) [inline fec/data/templates/partials/browse-data/candidates.jinja]: Same as above "Individual contribution totals by state" should be "Individual contributions by state"  We don't need "totals."

**Comment by patphongs** [comment]
@PaulClark2 Changes have been made for the contribution titles. Please check again, thanks!

**Pull request merged**